dune/stuff/common/color.cc

```diff
@@ -73,22 +76,22 @@ std::map< std::string, std::string >& color_map()
   static std::map< std::string, std::string > map_;
   static bool created_ = false;
   if (!created_) {
-    map_["black"]       = Colors::black;
-    map_["blue"]        = Colors::blue;
-    map_["brown"]       = Colors::brown;
-    map_["cyan"]        = Colors::cyan;
-    map_["darkgray"]    = Colors::darkgray;
-    map_["green"]       = Colors::green;
-    map_["lightblue"]   = Colors::lightblue;
-    map_["lightcyan"]   = Colors::lightcyan;
-    map_["lightgray"]   = Colors::lightgray;
-    map_["lightgreen"]  = Colors::lightgreen;
+    map_["black"] = Colors::black;
+    map_["blue"] = Colors::blue;
+    map_["brown"] = Colors::brown;
+    map_["cyan"] = Colors::cyan;
+    map_["darkgray"] = Colors::darkgray;
+    map_["green"] = Colors::green;
+    map_["lightblue"] = Colors::lightblue;
+    map_["lightcyan"] = Colors::lightcyan;
+    map_["lightgray"] = Colors::lightgray;
+    map_["lightgreen"] = Colors::lightgreen;
```

Comment:
This is something that I do not like at all about clang-format. There is improved readability in those aligned assignments. Perhaps
```
AlignConsecutiveAssignments: true
```
will fix this but again clang 3.6.2 does not (yet?) understand this. This is not a show stopper for me.